draft-ietf-quic-qlog-h3-events.md

```diff
@@ -441,15 +441,15 @@ headers: [
   }
 ]
 ~~~
-{: #http-headersframe-ex title="HTTPHeadersFrame example"}
+{: #h3-headersframe-ex title="H3HeadersFrame example"}
 
 ~~~ cddl
-HTTPHeadersFrame = {
+H3HeadersFrame = {
     frame_type: "headers"
     headers: [* HTTPField]
 }
 ~~~
-{: #http-headersframe-def title="HTTPHeadersFrame definition"}
+{: #h3-headersframe-def title="H3HeadersFrame definition"}
 
 ~~~ cddl
 HTTPField = {
```

Comment:
Discussed on call. Rename to H3HTTPField. Dirty, but consistent.

Reply:
Discussed on call. Postpone merge until we have an equivalent transport PR to merge along and until #253 is merged.